### PR TITLE
Fix bug in knative raw handler

### PR DESCRIPTION
**Pull request by pwplusnick**
There was a typing bug that I neglected to commit for the Knative platform 'raw' handler. This corrects the typing issue.

### CODE REVIEW

File: core/actionProxy/owplatform/knative.py [hunk 1]
```diff
@@ -108,15 +108,15 @@ def preProcessHTTPContext(msg, valueData):
...
+            valueData['__ow_body'] = base64.b64encode(bodyStr.encode())
         valueData['__ow_query'] = flask.request.query_string
 
     namespace = ''
     if '__OW_NAMESPACE' in os.environ:
         namespace = os.getenv('__OW_NAMESPACE')
     valueData['__ow_user'] = namespace
     valueData['__ow_method'] = flask.request.method
-    valueData['__ow_headers'] = flask.request.headers
+    valueData['__ow_headers'] = { k: v for k, v in flask.request.headers.items() }
```
Comment (mrutkows) [inline core/actionProxy/owplatform/knative.py]: This seems less like a typo and more like an array conversion issue... could you clarify?

File: core/actionProxy/owplatform/knative.py [hunk 2]
```diff
@@ -239,21 +239,18 @@ def run(self):
                 message['init'] = createInitDataFromEnvironment()
                 dedicated_runtime = True
 
+            preProcessRequest(message)
```
Comment (mrutkows) [inline core/actionProxy/owplatform/knative.py]: good efficiency catch! (again not a typo) ;)

File: core/actionProxy/owplatform/knative.py (hunk 1 above)
Comment (pwplusnick) [reply core/actionProxy/owplatform/knative.py]: Typing as in type system. It was the last thing I tested and I forgot to commit before I pushed.

File: core/actionProxy/owplatform/knative.py (hunk 2 above)
Comment (pwplusnick) [reply core/actionProxy/owplatform/knative.py]: This is basically what should have been the last commit, but never got committed before the push.

**Pull request merged**